sdk/synapse/Azure.Analytics.Synapse.Artifacts/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-preview.12 (Unreleased)
+## 1.0.0-preview.12 (2021-11-09)
```

Comment:
@idear1203, @wonner, could I ask one of you to add the areas below to this changelog, based on the updates to the swagger files this month?